### PR TITLE
Update help version

### PR DESCRIPTION
**Pull request by Blackbaud-BrandonJones**
Helps resolve this issue
https://github.com/blackbaud/skyux-lib-help/issues/12

works in tandem with SKY UX 2 PR 
https://github.com/blackbaud/skyux2/pull/1180

### CODE REVIEW

**Comment by Blackbaud-SteveBrush** [comment]
Putting this PR on-hold because it loosely depends on https://github.com/blackbaud/skyux2/pull/1180. Once that is merged and released, we can come back to this.

File: src/app/app.component.scss [hunk 1]
```diff
@@ -1,3 +1,11 @@
 .skyux-app-loading {
   visibility: hidden;
 }
+
+/deep/
+.sky-modal-body-full-page {
+  // Hide the bb-Help-invoker when a full-page modal is present and the widget is closed.
+  #bb-help-container.closed > #bb-help-invoker {
+    display: none;
```
Comment (Blackbaud-SteveBrush) [inline src/app/app.component.scss]: @Blackbaud-BobbyEarl What are your thoughts on adding an integration test for this behavior?
https://github.com/blackbaud/skyux-builder/tree/master/e2e

My thoughts: if we add a test, we could catch any selector changes from either the widget or UX.

Comment (Blackbaud-BobbyEarl) [reply src/app/app.component.scss]: I agree @Blackbaud-SteveBrush.  I talked to @Blackbaud-BrandonJones.  I'm comfortable getting this into builder right now due time constraints.  He's added a task to his backlog to work on a test for this scenario, as well as a few other help related scenarios.

**Comment by Blackbaud-BrandonJones** [comment]
removing the `[HOLD]` flag as https://github.com/blackbaud/skyux2/pull/1180 has been updated to approved.

**Pull request merged**